### PR TITLE
Ticket 6 created fixed navigation bar

### PR DESCRIPTION
**Pull request by vricci518**
Created a fixed navigation bar and adjusted on all pages.
Adjusted paragraph spacing on About Us page

### CODE REVIEW

**Review by margeschrec — APPROVED** [review]
looks good

File: aboutus.php [hunk 1]
```diff
@@ -102,6 +101,7 @@
             </p>
         </div>
     </footer>
+    </div>
 </body>
 
 </html>
```
Comment (billythakidd04) [inline aboutus.php:107]: no newline at end of file

File: contact.php [hunk 2]
```diff
@@ -112,6 +113,7 @@
             </p>
         </div>
     </footer>
+    </div>
 </body>
 
 </html>
```
Comment (billythakidd04) [inline contact.php:119]: no newline at end of file

File: css/aboutus.css [hunk 3]
```diff
@@ -19,7 +24,7 @@
 .about p{
     width: 90%;
     margin: auto;
-    padding: 10px 0px 10px 0px;
+    padding: 5px 0px 5px 0px;
```
Comment (billythakidd04) [inline css/aboutus.css:27]: this can be reduced to just `5px auto;`

File: menu.php [hunk 4]
```diff
@@ -164,6 +161,7 @@
             </p>
         </div>
     </footer>
+    </div>
 </body>
 
-</html>
+</html>
```
Comment (billythakidd04) [inline menu.php:167]: no newline at end of file

File: css/contact.css [hunk 5]
```diff
@@ -1,3 +1,8 @@
+.maincontact {
+    margin-top: 220px;
+    height: auto;
+}
+
 .contactform{
     width: 75%;
     padding: 20px 40px 20px 40px;
```
Comment (billythakidd04) [inline css/contact.css:8]: can also be reduced to `20px 40px`

File: css/home.css [hunk 6]
```diff
@@ -1,4 +1,10 @@
+.mainhome {
+    margin-top: 230px;
+    height: auto;
+}
+
 #homephoto{
+
```
Comment (billythakidd04) [inline css/home.css:7]: extra line

File: faq.php [hunk 7]
```diff
@@ -164,5 +165,6 @@
         </p>
     </div>
 </footer>
+</div>
 </body>
 </html>
```
Comment (billythakidd04) [inline faq.php:170]: no newline at end of file

File: home.php [hunk 8]
```diff
@@ -94,5 +95,6 @@
             </p>
         </div>
     </footer>
+    </div>
 </body>
 </html>
```
Comment (billythakidd04) [inline home.php:100]: newline

File: menu.php (hunk 4 above)
Comment (billythakidd04) [inline menu.php:167]: newline

File: socialmedia.php [hunk 9]
```diff
@@ -1,69 +1,74 @@
...
 <html>
-    <head>
-        <title>BAR-BILL TAVERN - Bar-Bill Tavern - Restaurant in NY</title>
-        <link rel="stylesheet" href="css\barbill.css">
-        <link rel="stylesheet" href="css\socialmedia.css">
+
+<head>
+    <title>BAR-BILL TAVERN - Bar-Bill Tavern - Restaurant in NY</title>
+    <link rel="stylesheet" href="css\barbill.css">
+    <link rel="stylesheet" href="css\socialmedia.css">
```
Comment (billythakidd04) [inline socialmedia.php:7]: the `\` should be `/`

File: socialmedia.php [hunk 10]
```diff
@@ -1,69 +1,74 @@
...
-                8326 Main Street<br />
-                Clarence, NY 14221<br />
-                (716) 710-9464
-            </p>
-        </div>
-    </footer>
-    </body>
+</body>
+
 </html>
```
Comment (billythakidd04) [inline socialmedia.php:74]: newline

**Review by billythakidd04 — COMMENTED** [review]
need to watch out for newlines at the end of the file, some parsers only know then end of the file by having a single blank line. Also be careful to use `/` path separators.

File: aboutus.php (hunk 1 above)
Comment (vricci518) [reply aboutus.php:107]: Fixed

File: contact.php (hunk 2 above)
Comment (vricci518) [reply contact.php:119]: Fixed

File: css/aboutus.css (hunk 3 above)
Comment (vricci518) [reply css/aboutus.css:27]: Adjusted

File: menu.php (hunk 4 above)
Comment (vricci518) [reply menu.php:167]: fixed

File: css/contact.css (hunk 5 above)
Comment (vricci518) [reply css/contact.css:8]: Adjusted

File: faq.php (hunk 7 above)
Comment (vricci518) [reply faq.php:170]: Fixed

File: home.php (hunk 8 above)
Comment (vricci518) [reply home.php:100]: Fixed

File: menu.php (hunk 4 above)
Comment (vricci518) [reply menu.php:167]: Fixed

File: socialmedia.php (hunk 9 above)
Comment (vricci518) [reply socialmedia.php:7]: Fixed

File: socialmedia.php (hunk 10 above)
Comment (vricci518) [reply socialmedia.php:74]: Fixed

**Comment by vricci518** [comment]
The New Line at the end of file is being removed when VS Code performs Format Document.
The issue with the \ vs / is happening on Margaret's machine. She enters a / but after saving an reopening it appears as \

**Comment by billythakidd04** [comment]
> The New Line at the end of file is being removed when VS Code performs Format Document.
> The issue with the \ vs / is happening on Margaret's machine. She enters a / but after saving an reopening it appears as \

Actually both issues are from vs code. Some small config changes we need to make. Need to tell it to use linux style path and `LF` line endings. I'm not sure if there's a way to do this globally but I will look and let you know.

**Pull request merged**